apps/win32/package.json

```diff
@@ -35,28 +35,27 @@
     "@office-iss/react-native-win32": "0.62.0-preview.3",
     "@office-iss/rex-win32": "0.62.0-preview.3-tenantreactnativewin-12906",
     "@react-native-community/eslint-config": "^1.1.0",
+    "@types/jasmine": "3.5.10",
     "@types/prop-types": "15.5.1",
     "@types/react": "^16.9.34",
     "@types/react-native": "^0.62.0",
     "@uifabricshared/build-native": "^0.1.1",
     "@uifabricshared/eslint-config-rules": "^0.1.1",
-    "metro-react-native-babel-preset": "^0.58.0",
-    "react": "16.11.0",
-    "react-native": "0.62.2",
-    "react-native-svg-transformer": "^0.14.3",
-    "react-test-renderer": "~16.11.0",
-    "appium": "1.17.1",
+    "@wdio/appium-service": "^5.0.0",
     "@wdio/cli": "^5.22.4",
     "@wdio/jasmine-framework": "^5.18.6",
     "@wdio/local-runner": "^5.22.4",
     "@wdio/spec-reporter": "^5.22.4",
     "@wdio/sync": "^5.20.1",
-    "@wdio/dot-reporter": "^5.22.4",
-    "@wdio/appium-service": "^5.0.0",
+    "appium": "1.17.1",
+    "metro-react-native-babel-preset": "^0.58.0",
+    "react": "16.11.0",
```

Comment:
do react and react-native need to be specified as devdeps if they're already normal dependencies?

Reply:
I don't believe so, testing now and removing in newest commit

Reply:
Removed the devdeps